src/debian/changelog

```diff
@@ -1,3 +1,15 @@
+python-fingerprint (1.5) UNRELEASED; urgency=medium
+
+  * Added support for Python3
+    * Still compatible with Python2
+  * storeTemplate() can be used without arguments
+    * The template is stored at the next free index automatically
+    * BREAKING CHANGE: The index is returned! In previous releases this was a boolean value.
+  * Introduced getStorageCapacity() to retrieve the sensor template capacity
+  * Introduced generateRandom() which generates a random number on the builtin hardware PRNG of the ZFM
+
+ -- Philipp Meisberger <team@pm-codeworks.de>  Sun, 09 Apr 2017 21:46:27 +0200
```

Comment:
My UTC is currently 16:48. Standard time is 18:48, so UTC +2 hours is correctly ;) I did not insert this time manually: "dch" program did it^^